### PR TITLE
Add A-Resistor

### PR DESCRIPTION
**Pull request by PraveenElango**
Adds a resistor component object with a value for resistance and SI unit

### CODE REVIEW

File: src/main/java/seedu/duke/component/Resistor.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public Resistor(double value){
+        super(value);
+    }
+    /**
+     * Returns the value and unit of Capacitor object.
+     *
+     * @return String of Capacitor value
+     */
+    @Override
+    public String toString(){
```
Comment (sevenseasofbri) [inline src/main/java/seedu/duke/component/Resistor.java]: `public String toString() {`

File: src/main/java/seedu/duke/component/Resistor.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package seedu.duke.component;
+
+public class Resistor extends Component {
+    private static final String SI_UNIT = "μH";
+
+    public Resistor(double value){
```
Comment (sevenseasofbri) [inline src/main/java/seedu/duke/component/Resistor.java]: `public Resistor(double value) {`

File: src/main/java/seedu/duke/component/Resistor.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        super(value);
+    }
+    /**
+     * Returns the value and unit of Capacitor object.
+     *
+     * @return String of Capacitor value
```
Comment (sevenseasofbri) [inline src/main/java/seedu/duke/component/Resistor.java]: Consider changing this to Resistor object

**Review by sevenseasofbri — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/component/Resistor.java (hunk 2 above)
Comment (PraveenElango) [reply src/main/java/seedu/duke/component/Resistor.java]: Thank you for pointing out!

File: src/main/java/seedu/duke/component/Resistor.java (hunk 3 above)
Comment (PraveenElango) [reply src/main/java/seedu/duke/component/Resistor.java]: Thank you for pointing out!

File: src/main/java/seedu/duke/component/Resistor.java (hunk 1 above)
Comment (PraveenElango) [reply src/main/java/seedu/duke/component/Resistor.java]: Thank you for pointing out!

**Comment by hughjazzman** [comment]
You don't need to close the PR and make a new one once you've made the changes. But it's okay, now that you've opened #19 we'll use that.

**Comment by PraveenElango** [comment]
@hughjazzman Got it, thanks for the help!

**Pull request closed without merging**